.pre-commit-config.yaml

```diff
@@ -1,6 +1,13 @@
 default_language_version:
-  python: "3.10"
```

Comment:
Why would that not work? It was intentionally set to 3.10.

Reply:
Encountered this:

```
$ pre-commit run
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/python3', '-mvirtualenv', '/home/ang/.cache/pre-commit/repor_ggvccg/py_env-3.10', '-p', '3.10')
return code: 1
stdout:
    RuntimeError: failed to find interpreter for Builtin discover of python_spec='3.10'
    
stderr: (none)
Check the log at /home/ang/.cache/pre-commit/pre-commit.log
```

See https://github.com/pre-commit/pre-commit/issues/1375

Should I revert?

Reply:
Ah ok - that is locally, of course. Yes, you are right, that makes sense - thanks!